docs/source/module_configuration/staticsite.rst

```diff
@@ -43,71 +305,62 @@ Incorporating CloudFront with a static site is a common best practice, however,
-              staticsite_web_acl: arn:aws:waf::123456789012:webacl/...
+1. The viewer’s web browser is redirected to Amazon Cognito custom UI page to sign up and authenticate.
+2. After authentication, Cognito generates and cryptographically signs a JWT then responds with a redirect containing the JWT embedded in the URL.
+3. The viewer’s web browser extracts JWT from the URL and makes a request to private content (private/* path), adding Authorization request header with JWT.
+4. Amazon CloudFront routes the request to the nearest AWS edge location. The CloudFront distribution’s private behavior is configured to launch a `Lambda@Edge` function on ViewerRequest event.
+5. `Lambda@Edge` decodes the JWT and checks if the user belongs to the correct Cognito User Pool. It also verifies the cryptographic signature using the public RSA key for Cognito User Pool. Crypto verification ensures that JWT was created by the trusted party.
+6. After passing all of the verification steps, `Lambda@Edge` strips out the Authorization header and allows the request to pass through to designated origin for CloudFront. In this case, the origin is the private content Amazon S3 bucket.
+7. After receiving response from the origin S3 bucket, CloudFront sends the response back to the browser. The browser displays the data from the returned response.
 
-              # staticsite_enable_cf_logging defaults to true
-              staticsite_enable_cf_logging: true
+An example of a `Auth@Edge`_ static site configuration is as follows. All listed options are required:
 
-              # Deploy Lambda@Edge to rewrite directory indexes
-              # e.g. support accessing example.org/foo/
-              staticsite_rewrite_directory_index: index.html
+.. code-block:: yaml
 
-              # You can also deploy custom Lambda@Edge associations with your
-              # pre-built function versions
-              # (this takes precedence over staticsite_rewrite_directory_index)
-              staticsite_lambda_function_associations:
-                - type: origin-request
-                  arn: arn:aws:lambda:us-east-1:123456789012:function:foo:1
+  deployments:
+    - modules:
+      - path: sample-app
+        type: static
+        parameters:
+          dev:
+            namespace: sample-app-dev
+            staticsite_auth_at_edge: true
+            staticsite_user_pool_arn: arn:aws:cognito-idp:us-east-1:240134083525:userpool/us-east-1_cjVgcUyWV
+      regions:
+        # NOTE: At the time of this writing Lambda@EDGE is only
+        # available in us-east-1. This solution will fail in
+        # other regions
+        - us-east-1
 
-              # Custom error response options can be defined
-              staticsite_custom_error_responses:
-                - ErrorCode: 404
-                  ResponseCode: 200
-                  ResponsePagePath: /index.html
+The `Auth@Edge`_ functionality does not create a Cognito User Pool for you.
```

Comment:
I don't love the language here (I promise I'll stop with the doc preference nitpicking now) -- would rather the emphasis was on the automatic management of the app client and not the manual user pool creation.

Could be something along the lines of "the auth@edge functionality uses your existing Cognito user pool..."

Reply:
Nit has been picked